repository/x.json

```diff
@@ -186,8 +186,12 @@
 			"labels": ["xonsh", "python", "language syntax"],
 			"releases": [
 				{
-					"sublime_text": ">=4075",
-					"tags": true
+					"sublime_text": "4075 - 4147",
+					"tags": "4075-4147_"
+				},
+				{
+					"sublime_text": ">=4148",
+					"tags": "4148_"
```

Comment:
This looks a bit impractical to me. You should be able to continue your normal numbering for 4148+ and only serve prefixed tags to legacy versions.

Reply:
but then in the future if another release requires another version, I'd need to either retag all those versions or make all the future versions from that point on get another prefix? So then the benefit of non-prefixed versions is temporary anyway

Reply:
no, you won't have to re-tag existing versions. just introduce a prefixed release, pinning certain versions of ST on that release. Ie. you use prefixed releases to pin legacy versions while your ongoing releases are always simply ongoing numbers.

Reply:
Ok, I've updated the format to just `"tags":true`, but only `>=4148`, so the non-prefixed tags like `0.2.1` should be picked up just by the new version

Reply:
Hm, I see the package is updated to 0.2.1, but then this is the version that is installed even on older ST versions (checked with 4134)